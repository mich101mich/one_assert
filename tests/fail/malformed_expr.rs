fn main() {
    one_assert::assert!(1 ==);
    one_assert::assert!(1 ==, "hi");
    one_assert::assert!(== 1);
    one_assert::assert!(== 1, "hi");
    one_assert::assert!(==);
    one_assert::assert!(==, "hi");
}
