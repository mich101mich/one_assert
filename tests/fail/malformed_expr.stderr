error: incomplete expression: unexpected end of input, expected an expression
 --> tests/fail/malformed_expr.rs:2:25
  |
2 |     one_assert::assert!(1 ==);
  |                         ^^^^

error: Expression before the comma is incomplete: expected an expression
 --> tests/fail/malformed_expr.rs:3:29
  |
3 |     one_assert::assert!(1 ==, "hi");
  |                             ^

error: expected an expression
 --> tests/fail/malformed_expr.rs:4:25
  |
4 |     one_assert::assert!(== 1);
  |                         ^

error: expected an expression
 --> tests/fail/malformed_expr.rs:5:25
  |
5 |     one_assert::assert!(== 1, "hi");
  |                         ^

error: expected an expression
 --> tests/fail/malformed_expr.rs:6:25
  |
6 |     one_assert::assert!(==);
  |                         ^

error: expected an expression
 --> tests/fail/malformed_expr.rs:7:25
  |
7 |     one_assert::assert!(==, "hi");
  |                         ^
